tools/adventure-pack/goodies/typescript/Iterator.prototype.toArray/index.test.ts

```diff
@@ -0,0 +1,24 @@
+import "../Iterator.prototype.toArray";
```

Comment:
I think in the other tests I've been trying to write it as `import "./index";`

Reply:
Ack!